The stylelint-disable-line command fails when used within a value list

> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug where `color-named` reports color keywords like `red` or `green` within sass maps.
This behaviour can't be deactivated, even with stylelint-disable-line comments.

> Which rule, if any, is this issue related to?

`color-named`

> What CSS is needed to reproduce this issue?

```scss
$colors: (
  'error': red,
);
```

as well as all the following disable comments:

```scss
$colors: (
  'error': red, /* stylelint-disable-line color-named */
  'error': red, // stylelint-disable-line color-named
  'error': red, // stylelint-disable-line
  'error': red, /* stylelint-disable-line */
);
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "color-named": "never"
  }
}
```

> Which version of stylelint are you using?

`6.8.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

with `gulp-stylelint`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, it's related to SCSS maps.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
app/util/color/vars.scss
  52:14  warning  Unexpected named color "red"     color-named
```

--- a/docs/user-guide/configuration.md
+++ b/docs/user-guide/configuration.md
@@ -95,6 +95,8 @@ Complex, overlapping disabling & enabling patterns are supported:
 /* stylelint-enable foo */
 ```
 
+**Caveat:** Comments within *selector and value lists* are currently ignored.
+
 #### Severities: error & warning
 
 By default, all rules have an `"error"`-level severity. You can change this default by adding a `defaultSeverity` property to your configuration (see below).

--- a/src/rules/comment-empty-line-before/README.md
+++ b/src/rules/comment-empty-line-before/README.md
@@ -14,6 +14,8 @@ If the comment is the very first node in a stylesheet then it is ignored. Shared
 
 If you're using a custom syntax which support single-line comments with `//`, those are ignored as well.
 
+**Caveat:** Comments within *selector and value lists* are currently ignored.
+
 ## Options
 
 `string`: `"always"|"never"`

--- a/src/rules/comment-no-empty/README.md
+++ b/src/rules/comment-no-empty/README.md
@@ -8,6 +8,8 @@ Disallow empty comments.
  * Comments like this */
 ```
 
+**Caveat:** Comments within *selector and value lists* are currently ignored.
+
 ## Options
 
 ### `true`

--- a/src/rules/comment-whitespace-inside/README.md
+++ b/src/rules/comment-whitespace-inside/README.md
@@ -10,6 +10,8 @@ Require or disallow whitespace on the inside of comment markers.
 
 Any number of asterisks are allowed at the beginning or end of the comment. So `/** comment **/` is treated the same way as `/* comment */`.
 
+**Caveat:** Comments within *selector and value lists* are currently ignored.
+
 ## Options
 
 `string`: `"always"|"never"`

--- a/src/rules/comment-word-blacklist/README.md
+++ b/src/rules/comment-word-blacklist/README.md
@@ -8,6 +8,8 @@ Specify a blacklist of disallowed words within comments.
  * These three words */
 ```
 
+**Caveat:** Comments within *selector and value lists* are currently ignored.
+
 ## Options
 
 `array|string`: `["array", "of", "words", "or", "/regex/"]|"word"|"/regex/"`

